src/container-instance.class.mts

```diff
@@ -1494,35 +1495,6 @@ export class ContainerInstance implements Disposable {
     }
   }
 
-  private resolveTypeWrapper(wrapper: TypeWrapper): ServiceIdentifier {
```

Comment:
The removal of this, even with it being private, should be in the changeset somewhere...